src/software/thunderscope/thunderscope_main.py

```diff
@@ -13,7 +13,7 @@
 import proto.message_translation.tbots_protobuf as tbots_protobuf
 from software.thunderscope.robot_communication import RobotCommunication
 from software.thunderscope.replay.proto_logger import ProtoLogger
-from software.thunderscope.constants import EstopMode, ProtoUnixIOTypes
+from software.thunderscope.constants import EstopMode, ProtoUnixIOTypes, TabNames
```

Comment:
is this import needed?

Reply:
You've found another artifact from an ancient time - :1st_place_medal: